doc/source/examples/dem/plate-discharge/plate-discharge.rst

```diff
@@ -0,0 +1,496 @@
+
+
+Angle of Repose
+~~~~~~~~~~~~~~~
+
+The angle of repose is calculated from the data extracted from the VTU output files. Angles of repose are calculated from the pile of particles on the plate for comparison with the literature, and from the piles formed by the discharge for curiosity.
+
+The configuration of the case gives a symmetrical formation of the piles, meaning that there are 2 angles of repose to calculate over and below the plate.
+In order to show how the results may fluctuated in regards of that, we show the angle obtained from the particle positions from the left and the right sides.
+
+
+The angles of repose are calculated by linear regressions from the highest particle positions in y-axis from -0.35 m to -0.15 m for the left angles and from 0.15 m to 0.35 m for the right angles in x-axis. The following figure shows the areas where the angles are calculated. The given angles of reposes are the linear regressions from the positions with absolute x coordinates.
+
+.. figure:: images/angle-areas.png
+   :alt: Angle of repose areas
+   :align: center
+   :name: plate-discharge-angle-areas
+
+   The areas where the angle of repose is calculated for the left (blue) and right (red) sides of the piles.
+
+.. note::
+   The calculated angles ignore the wall effects since all the particles in the depth of the container are taken into account.
+
+According to Zhou *et al.* [#zhou2002]_, the angle of repose from this type of configuration is calculated with the following formula:
+
+.. math::
+   \phi = 68.61 \mu_{\text{f,pp}}^{0.27} \mu_{\text{f,pw}}^{0.22} \mu_{\text{r,pp}}^{0.06} \mu_{\text{r,pw}}^{0.12} d_p^{-0.2}
+
+
+where :math:`\mu_{\text{f,pp}}` and :math:`\mu_{\text{f,pw}}` are the friction coefficients of the particle-particle and particle-wall interactions, respectively, :math:`\mu_{\text{r,pp}}` and :math:`\mu_{\text{r,pw}}` are the rolling friction coefficients, and :math:`d_p` is the particle diameter.
+
+
+The meaning of the rolling friction coefficient by the authors [#zhou2002]_ is different than found in Lethe. They express the coefficient as aa length in the `rolling friction model <../../../theory/multiphase/cfd_dem/dem.html#rolling-friction-models>`_. However, they also use the constant torque, therefore the rolling friction coefficient in Lethe as to be multiplied by the effective radius of the particle for results comparison:
+
+.. math::
+   \mu_{\text{r}}^{\text{eqt}} = \mu_{\text{r}}^{\text{lethe}}D_p
+
+.. figure:: images/angle-of-repose.png
+   :alt: Angle of repose results
+   :align: center
+   :name: plate-discharge-angle-graph
+
+   The angles of repose calculated from the simulation data. The solid lines are the angles computed from the highest particles on both side, while the shaded areas represent the angles for the left and the right.
```

Comment:
I don't understand the difference between the solid lines and the shaded area. The solid line is the angle on the plate?

Reply:
A pile has 2 "symmetrical" angles, so the solid lines are the linear regressions of the angles from the 2 sides and the shaded areas are the linear regressions from one (left or the right) angle.
I tried to rephrase  in the doc, but this is not obvious